sdks/java/core/src/main/java/com/google/cloud/dataflow/sdk/io/BigQueryIO.java

```diff
@@ -501,6 +502,21 @@ private static void dryRunQuery(BigQueryOptions options, String query) {
         return TableRowJsonCoder.of();
       }
 
+      @Override
+      public void populateDisplayData(DisplayData.Builder builder) {
+        if (table != null) {
+          builder.add("tableSpec", toTableSpec(table));
+        }
+
+        if (query != null) {
+          builder.add("query", query);
+        }
+
+        if (flattenResults != null) {
+          builder.add("flattenResults", flattenResults);
+        }
```

Comment:
This was a common thread of feedback from other PR's as well-- I'll add some constructs to register values if not default.